sheet_to_triples/trans.py

```diff
@@ -37,6 +37,7 @@ class _Converter:
     def __init__(self, reference_graph):
         self.vformat = string.Formatter().vformat
         self.resolver = reference_graph.store.namespace
+        self.from_identifier = rdf.Resolver(reference_graph).from_identifier
```

Comment:
This is weird, I don't think I like it. At the very least make it an underscore flagged private?

Is resolver still needed?

Reply:
`self.resolver` is still needed as `from_qname` requires it and I've not yet moved that from being a separate function to a method on the new `Resolver` class. I've refactored to make it closer to where I want the code to be once that change has been made - `self.resolver` is now our `Resolver` class, and the resolver used by `from_qname` is now `self._old_resolver` with a TODO saying it should be deprecated.